mypy.ini

```diff
@@ -131,18 +131,12 @@ ignore_errors = True
 [mypy-torch.cuda]
 ignore_errors = True
 
-[mypy-torch._lobpcg]
-ignore_errors = True
-
 [mypy-torch._appdirs]
 ignore_errors = True
 
 [mypy-torch._utils]
 ignore_errors = True
 
-[mypy-torch._overrides]
-ignore_errors = True
```

Comment:
You removed this in another PR as well, that'll give a merge conflict. Would be best to remove this I think, and also double check that you didn't do `_lobpcg` in another PR.

Reply:
Thanks for the review, Ralf. I've checked the other PRs I have opened and none of them touched `_lobpcg`